env0/resource_team_environment_assignment.go

```diff
@@ -31,7 +31,7 @@ func resourceTeamEnvironmentAssignment() *schema.Resource {
 			},
 			"role_id": {
 				Type:             schema.TypeString,
-				Description:      "id of the assigned role",
+				Description:      "id of the assigned custom role. The following built-in roles can be passed as well: Viewer, Planner, Deployer, Admin",
```

Comment:
I'm not sure about the markup language here but if we can put those values rendered as code (like back ticks in github) that would be nicer